### DISCUSSION TITLE
灌水。法厄同落网啦？

### DISCUSSION THREAD

**Discussion by MaskedFools-Mochun** [share121/inter-knot#2776] [其他]
最近，流传出这样一张图片。
![20240816161351](https://github.com/user-attachments/assets/9556be87-5906-4f4e-975c-5a9f5fa6a0a1)
图中男子自称传奇绳匠法厄同，位置看样子是在治安局门前，身后还有俩美女治安官。应该就是被这俩抓住的吧？啧啧啧，各位绳匠以后出门可得小心了。唉，这年头当绳匠钱可真难挣啊。

**Comment by lllliufeng** [comment]
营销号？何时来的

**Comment by AnkaAlioth** [comment]
治安局里自拍，这就是传奇大前辈的从容吗<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/651fba00ba55004182fffbb5aa553c53_6086077287436441482.png" alt="米仔_膜拜" width="75" >

**Comment by wxgl** [comment]
![3485034E30C199C1A4900C015C622F79_0](https://github.com/user-attachments/assets/da10fed2-cd8d-41e3-a7fd-e47bea807b65)

**Comment by demolute** [comment]
请问是星穹列车的无名客星女士吗

**Reply by MaskedFools-Mochun** [reply]
woc！盒！

**Reply by bushigemengsishuihaiyinga** [reply]
我超，盒！爱……星错了，星再也不敢在网上口嗨了

**Comment by AnonCN** [comment]
tql

**Comment by haxyyx** [comment]
怎么又有营销号，法厄同不是两个人吗，这件事在绳网人尽皆知吧，而且传奇绳匠怎么可能被治安局抓到，一点消息都没有。

**Comment by RaphaelHider** [comment]
一眼丁假，这图P的毫无水准，谁信我笑谁一辈子

**Comment by Lyfer123** [comment]
这个委托我接了！

**Comment by Manbow1997** [comment]
听说治安局的朱鸢警长在和法厄同约会？这是真的，前几天和朋友去吃火锅，我还看到两个人了，举止亲昵，有说有笑的。

**Comment by 56712354** [comment]
不是，楼主快叫照片里那人看我委托
否则他今晚就别回来了。
(卡在墙里的铃)
